line-bot-api-client/src/main/java/com/linecorp/bot/client/LineBlobClientImpl.java

```diff
@@ -85,12 +85,20 @@ public void onFailure(final Call<ResponseBody> call, final Throwable t) {
         }
 
         private MessageContentResponse convert(final Response<ResponseBody> response) {
+            ResponseBody body = response.body();
+            if (body == null) {
```

Comment:
I guess you can use [Objects.requireNonNull()](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#requireNonNull-T-java.lang.String-)
(but not sure spotbugs can detect that)

Reply:
good catch. i'll try it.